analyzer/src/main/kotlin/PackageJsonUtils.kt

```diff
@@ -107,7 +120,16 @@ internal class PackageJsonUtils {
         }
 
         private fun getWorkspaceMatchers(definitionFile: File): List<PathMatcher> {
-            var workspaces = definitionFile.readValue<ObjectNode>()["workspaces"]
+            var workspaces = try {
+                definitionFile.readValue<ObjectNode>()["workspaces"]
+            } catch (e: JsonProcessingException) {
+                e.showStackTrace()
+
+                log.error { "Could not parse '${definitionFile.invariantSeparatorsPath}': ${e.message}" }
+
+                return emptyList()
```

Comment:
nit: maybe re-using below code path by assigning `workspaces` to null in case of an exception is slightly better.

Reply:
Done.